source/docs/03-metadata.xml

```diff
@@ -1308,6 +1310,7 @@
                   </specList>
                </p>
                <p>The content of the <gi scheme="MEI">item</gi> is quite similar to <gi scheme="MEI">manifestation</gi> with a few omissions:</p>
+               <p>Please note that in case of handwritten sources the <att>singleton</att> attribute is available for the <gi scheme="MEI">manifestation</gi> element to indicate a unique physical object, precisely, manifestation singleton are sources <quote>produced as unique objects, with no siblings <hi rend="italic">intended</hi> in the course of their production.</quote> (<ref target="https://repository.ifla.org/handle/123456789/659">FRBRoo</ref>, p. 57). These cases result in <gi scheme="MEI">item</gi> and <gi scheme="MEI">manifestation</gi> being essentially the same.</p>
```

Comment:
```suggestion
               <p>Please note that in case of handwritten sources the <att>singleton</att> attribute is available for the <gi scheme="MEI">manifestation</gi> element to indicate a unique physical object, precisely, manifestation singleton are sources <quote>produced as unique objects, with no siblings intended in the course of their production.</quote> (<ref target="https://repository.ifla.org/handle/123456789/659">Definition of FRBRoo</ref>, p. 57). These cases result in <gi scheme="MEI">item</gi> and <gi scheme="MEI">manifestation</gi> being essentially the same.</p>
```

Reply:
Thanks @ahankinson. Yeah you're right that the italics are not part of the direct quote, however, I thought it might be good to add some emphasis because I read the "intended" as an important word in the sentence. Personally I like the idea of havin full bibliographic references but this might be better adressed in a different PR?

Reply:
I think it doesn't lose anything without the emphasis. However, if you do want, you should add "(emphasis added)" to the quote to indicate that it wasn't in the original -- otherwise, you're actually misquoting since you're implying a meaning that may or may not be there in the original.

A quick look suggests that doesn't seem to be any real standards for citation, so I guess we should add it later...